wrapper/src/test/java/com/moodysalem/java/TestPhantomJS.java

```diff
@@ -38,20 +36,21 @@ public void testRunScript() throws IOException, InterruptedException {
 
     @Test
     public void testOptions() throws IOException {
-        assertTrue(PhantomJS.exec(TestPhantomJS.class.getResourceAsStream("test-script.js"), PhantomJSOptions.DEFAULT.withHelp(true)).getExitCode() == 0);
-        assertTrue(PhantomJS.exec(TestPhantomJS.class.getResourceAsStream("test-script.js"), PhantomJSOptions.DEFAULT.withVersion(true)).getExitCode() == 0);
+        assertTrue(PhantomJS.exec(TestPhantomJS.class.getResourceAsStream("test-script.js"), PhantomJsOptions.DEFAULT.withHelp(true)).getExitCode() == 0);
+        assertTrue(PhantomJS.exec(TestPhantomJS.class.getResourceAsStream("test-script.js"), PhantomJsOptions.DEFAULT.withVersion(true)).getExitCode() == 0);
```

Comment:
because PhantomJsOptions is mutable in this PR, the meaning of this test has changed-now it calls help twice